src/SB/Game/zMain.h

```diff
@@ -1,4 +1,34 @@
 #ifndef ZMAIN_H
 #define ZMAIN_H
 
+#include <types.h>
+#include "xIni.h"
+#include "xserializer.h"
+
+st_SERIAL_PERCID_SIZE* g_xser_sizeinfo;
+
+
+enum eStartupErrors {
+    eNoError = 0,
+    eNoFormat = 1,
+    eDamagedCard = 2,
+    eWrongDevice = 3,
+    eNoCards = 4,
+    eCorruptFile = 5,
+    eNoController = 6,
```

Comment:
Since the default mode for enums is starting at value 0, it's more concise to define an enum like this by omitting the explicit values. Most IDEs will show you the integer value if you hover over the enum value. I'll make note of this one for some style guide discussion with the maintenance team. We can leave it **as is**, for now.

To clarify, writing it way does make it easier to decompile usages of the enum, which is a certain benefit of writing it this way.
However, most of our tooling already handles mapping from symbol name to value (Ghidra, m2c), and the values clutter things visually for later down the line when folks will be modifying or otherwise editing these values.